### PR TITLE
Feat/add additional version properties

### PR DESCRIPTION
**Pull request by tanviranadive**

### CODE REVIEW

File: internal/model/kbom.go [hunk 1]
```diff
@@ -58,10 +58,15 @@ type Components struct {
...
 type Resource struct {
-	Kind       string `json:"kind,omitempty"`
-	APIVersion string `json:"api_version,omitempty"`
-	Name       string `json:"name"`
-	Namespace  string `json:"namespace,omitempty"`
+	Kind                 string                `json:"kind,omitempty"`
+	APIVersion           string                `json:"api_version,omitempty"`
+	Name                 string                `json:"name"`
+	Namespace            string                `json:"namespace,omitempty"`
+	AdditionalProperties *AdditionalProperties `json:"additional_properties,omitempty"`
```
Comment (mateuszdyminski) [inline internal/model/kbom.go]: Can we change `AdditionalProperties` to `map[string]string` to make it more flexible?

File: internal/utils/utils.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package utils
 
 import (
 	"fmt"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```
Comment (mateuszdyminski) [inline internal/utils/utils.go]: can we keep imports into 3 blocks separated by empty line: 

```
std imports

external imports

imports from github.com/rad-security/kbom/
```

File: internal/kube/kube.go [hunk 3]
```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/sha256"
 	"fmt"
+	"github.com/rad-security/kbom/internal/utils"
```
Comment (mateuszdyminski) [inline internal/kube/kube.go]: can we keep imports into 3 blocks separated by empty line: 

```
std imports

external imports

imports from github.com/rad-security/kbom/
```

File: internal/utils/utils.go [hunk 4]
```diff
@@ -24,3 +25,19 @@ func BindFlags(cmd *cobra.Command) {
 		}
 	})
 }
+
+func GetVersion(item unstructured.Unstructured) (version string, ok bool) {
```
Comment (mateuszdyminski) [inline internal/utils/utils.go]: if it's used only in 1 place - let's do not move it to `utils`

we could keep it in: `internal/kube/kube.go` file

File: internal/model/kbom.go (hunk 1 above)
Comment (tedd-E) [reply internal/model/kbom.go]: Updated data structure to `map[string]string`

File: internal/utils/utils.go (hunk 2 above)
Comment (tedd-E) [reply internal/utils/utils.go]: Sorted imports

File: internal/kube/kube.go (hunk 3 above)
Comment (tedd-E) [reply internal/kube/kube.go]: Sorted imports

File: internal/utils/utils.go (hunk 4 above)
Comment (tedd-E) [reply internal/utils/utils.go]: Refactored function to `kube.go`

**Comment by tedd-E** [comment]
Hi @mateuszdyminski, please let us know if any further changes are required for this PR and in #135, thanks!

**Review by mateuszdyminski — APPROVED** [review]

**Review by jeffreyfriedman — APPROVED** [review]

**Comment by jeffreyfriedman** [comment]
@tanviranadive I think the Github Actions have have gotten into a bad state.  Would you mind pushing a trivial commit to try and kick them back into operation?

**Pull request merged**